rel/app/env.sh.bat

```diff
@@ -1 +1,2 @@
 set RELEASE_MODE=interactive
+set LIVEBOOK_ROOT_PATH=%HOMEPATH%
```

Comment:
```suggestion
set LIVEBOOK_ROOT_PATH=%USERPROFILE%
```

I think that's the preferred nowadays. If using HOMEPATH, you would also need HOMEDRIVE. Thoughts?